main.py

```diff
@@ -46,10 +46,10 @@ def createMsg(data):
     elif fileExtn == "zip" and fileType == "dd":
         msg["description"] = 'Data Delivery files uploaded to GCP bucket from Blaise5'
         msg["dataset"] = 'blaise_dde'
-        msg["iterationL1"] = os.getenv('ON-PREM-SUBFOLDER')
-        msg["iterationL2"] = data['name'][3:7].upper()
-        msg["iterationL3"] = data['name'][7:8].upper()
-        msg["iterationL4"] = ''
+        msg["iterationL1"] = 'SYSTEMS'
+        msg["iterationL2"] = os.getenv('ON-PREM-SUBFOLDER')
+        msg["iterationL3"] = data['name'][3:6].upper()
```

Comment:
Was this meant to change from 3:7 to 3:6?

If the name of the file start OPN2004 for example, won't that make iterationL3 200, not 2004?

Reply:
Just double checked, the Python : operator is exclusive of the last value, so that will only pull out 3 characters, not 4.